scripts/user/bsqs_client.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
```

Comment:
Looks like a Python client for the queueserver, used as a demonstration.  We can delete the entire file.